cdxev/auxiliary/io_processing.py

```diff
@@ -157,3 +158,31 @@ def create_list_file_filename(sbom: dict) -> str:
         )
         file = destination.open("w")
     file.write(list_file)
+
+
+def add_output_argument(parser: argparse.ArgumentParser) -> None:
+    """Helper function to create uniform output options for all commands."""
+    parser.add_argument(
+        "--output",
+        "-o",
+        metavar="<file>",
+        help=(
+            "The path to where the output should be written. If this is a file, output is "
+            "written there. If it's a directory, output is written to a file with an "
+            "auto-generated name inside that directory. If it's not specified, output is written "
+            "to stdout."
+        ),
+        type=Path,
+    )
+
+
+def add_input_argument(
+    parser: argparse.ArgumentParser,
+    nargs: str = "",
+    help: str = "Path to the SBOM file.",
+) -> None:
+    """Helper function to create uniform input options for all commands."""
+    if nargs:
+        parser.add_argument("input", metavar="input", help=help, type=Path, nargs=nargs)
+    else:
+        parser.add_argument("input", metavar="input", help=help, type=Path)
```

Comment:
The `metavar` should be "<input>", as it is a placeholder, which is indicated by the "<>"